pkg/controller/servicemesh/webhookca/source.go

```diff
@@ -31,14 +31,15 @@ type SecretCABundleSource struct {
 
 type SecretNameKeyPair struct {
 	SecretName string
-	Key        string
+	Keys       []string
 }
 
 var _ CABundleSource = (*SecretCABundleSource)(nil)
 
 func (s *SecretCABundleSource) GetCABundle(ctx context.Context, client client.Client) ([]byte, error) {
 	var caBundle []byte
 	var errList []error
+loop:
```

Comment:
maybe call this `outerLoop` since there are two loops?

Reply:
or extract the outer loop into a function and use `return` instead of `break`?

Reply:
Removed